CHECK_POWER_COMMANDS/Backstop_File_Class.py

```diff
@@ -266,4 +266,4 @@ def insert_errors( self, lr_file, violations_list):
             print('Copying ACIS-LoadReview.txt.ERRORS to ACIS-LoadReview.txt')
             mv_status = subprocess.run(['mv', 'ACIS-LoadReview.txt.ERRORS', 'ACIS-LoadReview.txt'])
```

Comment:
`shutil.move` is the correct way to move a file or directory in Python rather than OS-specific system calls.